### PR TITLE
Major changes: design guide content refactor

### PR DESCRIPTION
**Pull request by klepas**
## Description

This branch holds all the SASS comment documentation changes.

Most of the edits here are deletions, where documentation and guidance that previously was entirely contained within the SASS partials have been moved into the [Design Guide](https://github.com/AusDTO/dto-design-guide/), as markdown content.

These changes make the current KSS build system redundant.

We may also want to pursue further tidy-ups — expunge anything else that is no longer now being used, but that can continue after.

## Definition of Done

- [x] Content/documentation reviewed by Julian or someone in the Content Team
- [x] Code reviewed by one of the core developers
- [x] Acceptance Testing
  - [x] HTML5 validation (CircleCI)
  - [x] Accessibility testing & WCAG2 compliance (`npm test`)
- [ ] Stakeholder/PO review
- [x] CHANGELOG updated

### CODE REVIEW

**Comment by klepas** [comment]
No new errors from the `npm test` results.

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+[*]
+indent_style = space
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.md]
+trim_trailing_whitespace = false
```
Comment (dominikwilkowski) [inline .editorconfig:12]: finally!

File: .gitignore [hunk 2]
```diff
@@ -51,3 +51,10 @@ data-sections.json
 build
 
 .DS_Store
+
+### Sublime ###
+*.sublime-project
+*.sublime-workspace
+
+### Yarn ###
+yarn.lock
```
Comment (dominikwilkowski) [inline .gitignore:60]: uh yesssss

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,11 @@
 
 ## UI Kit "Kraken"
 
+### `develop`
+
+- Most of the guidance previously sitting in the SASS partials have gracefully uprooted and migrated themselves into the new [Design Guide repository](https://github.com/AusDTO/dto-design-guide/), stored now in Markdown. They like their new home.
```
Comment (dominikwilkowski) [inline CHANGELOG.md:9]: perhaps worth adding http://guides.service.gov.au/design-guide/ domain here?

**Review by dominikwilkowski — CHANGES_REQUESTED** [review]
are we sure we need those images in this repo? I'm almost certain that we moved them into the design-guide repo entirely?!

File: CHANGELOG.md (hunk 3 above)
Comment (klepas) [reply CHANGELOG.md:9]: Agreed.

**Comment by klepas** [comment]
@dominikwilkowski Done. :)

**Review by dominikwilkowski — APPROVED** [review]

**Pull request merged**